### DISCUSSION TITLE
Icon/color select based on tracker?

### DISCUSSION THREAD

**Discussion by ArneAnka** [transmission/transmission#5344] [Ideas]
Hi!

I download from a few selected trackers, and it can be a hassel to keep track of wich torrent is being download from what tracker. So it would be cool if it was possible to change the icon, or the icon color.

![SCR-20230402-jz0](https://user-images.githubusercontent.com/5988810/229352615-32eba18b-8901-4dde-a77a-979172e4793f.png)

<img width="394" alt="Skärmavbild 2023-04-02 kl  14 21 21" src="https://user-images.githubusercontent.com/5988810/229352517-70bb2011-2945-4ed0-923c-c4ea1bc694b4.png">

<img width="321" alt="Skärmavbild 2023-04-02 kl  14 18 27" src="https://user-images.githubusercontent.com/5988810/229352525-5392dbca-dff5-4410-b8db-45e4a39fe271.png">

**Comment by dantearmok** [comment]
Turn on Groups from the View menu.  Then visit the Groups prefs.  You can create a filter there, on a tracker name, etc.

**Reply by ArneAnka** [reply]
Thanks! But im using the webclient, possible?
